### ISSUE TITLE
Class 'Spatie\Mailcoach\Http\App\Queries\SendQuery' not found

### ISSUE THREAD

**Issue by ryancwalsh** [spatie/mailcoach-support#27]
When I'm at `/mailcoach/email-lists/1/subscriber/1/details` and click **`/mailcoach/email-lists/1/subscriber/1/received-campaigns`** I get this error that I haven't figured out yet:

```
/home/vagrant/Code/myproject/storage/framework/views/5805ff27c302dc3ac1d47f23cbf9b6e1704a2552.php:49:
object(Symfony\Component\Debug\Exception\FatalThrowableError)[1513]
  private 'originalClassName' => string 'Error' (length=5)
  protected 'message' => string 'Class 'Spatie\Mailcoach\Http\App\Queries\SendQuery' not found' (length=61)
  private 'string' (Exception) => string '' (length=0)
  protected 'code' => int 0
  protected 'file' => string '/home/vagrant/Code/myproject/vendor/spatie/laravel-mailcoach/src/Http/App/Controllers/EmailLists/Subscribers/ReceivedCampaignsController.php' (length=147)
  protected 'line' => int 14
  private 'trace' (Exception) => 
    array (size=45)
      0 => 
        array (size=5)
          'file' => string '/home/vagrant/Code/myproject/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php' (length=108)
          'line' => int 48
          'function' => string '__invoke' (length=8)
          'class' => string 'Spatie\Mailcoach\Http\App\Controllers\EmailLists\Subscribers\ReceivedCampaignsController' (length=88)
          'type' => string '->' (length=2)
      1 => 
        array (size=5)
          'file' => string '/home/vagrant/Code/myproject/vendor/laravel/framework/src/Illuminate/Routing/Route.php' (length=93)
          'line' => int 219
          'function' => string 'dispatch' (length=8)
          'class' => string 'Illuminate\Routing\ControllerDispatcher' (length=39)
          'type' => string '->' (length=2)
      2 => 
        array (size=5)
          'file' => string '/home/vagrant/Code/myproject/vendor/laravel/framework/src/Illuminate/Routing/Route.php' (length=93)
          'line' => int 176
          'function' => string 'runController' (length=13)
          'class' => string 'Illuminate\Routing\Route' (length=24)
          'type' => string '->' (length=2)
      3 => 
        array (size=5)
          'file' => string '/home/vagrant/Code/myproject/vendor/laravel/framework/src/Illuminate/Routing/Router.php' (length=94)
          'line' => int 681
          'function' => string 'run' (length=3)
          'class' => string 'Illuminate\Routing\Route' (length=24)
          'type' => string '->' (length=2)
      4 => 
        array (size=5)
          'file' => string '/home/vagrant/Code/myproject/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php' (length=97)
          'line' => int 130
          'function' => string 'Illuminate\Routing\{closure}' (length=28)
          'class' => string 'Illuminate\Routing\Router' (length=25)
          'type' => string '->' (length=2)
      5 => 
        array (size=5)
          'file' => string '/home/vagrant/Code/myproject/vendor/spatie/laravel-mailcoach/src/Http/App/Middleware/SetMailcoachDefaults.php' (length=116)
```

**Comment by ryancwalsh** [comment]
I'm also seeing a similar error at /mailcoach/campaigns/4/outbox (which I visited after my CampaignMailSentEventListener seemed to have a problem):

```
object(Symfony\Component\Debug\Exception\FatalThrowableError)[1459]
  private 'originalClassName' => string 'Error' (length=5)
  protected 'message' => string 'Class 'Spatie\Mailcoach\Http\App\Queries\SendsQuery' not found' (length=62)
  private 'string' (Exception) => string '' (length=0)
  protected 'code' => int 0
  protected 'file' => string '/home/vagrant/Code/myproject/vendor/spatie/laravel-mailcoach/src/Http/App/Controllers/Campaigns/Sent/OutboxController.php' (length=128)
  protected 'line' => int 12
  private 'trace' (Exception) => 
    array (size=45)
      0 => 
        array (size=5)
          'file' => string '/home/vagrant/Code/myproject/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php' (length=108)
          'line' => int 48
          'function' => string '__invoke' (length=8)
          'class' => string 'Spatie\Mailcoach\Http\App\Controllers\Campaigns\Sent\OutboxController' (length=69)
```

**Comment by freekmurze** [comment]
Thanks for reporting this.

Both these problems have been fixed in `v1.3.1` which I've tagged a minute ago.

**Comment by ryancwalsh** [comment]
@freekmurze I notice you working on the weekend (like I am, excited to get Mailcoach working) and really appreciate it! :-)
